verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -995,16 +995,19 @@ const TreeUnwrapperTestData kUnwrapModuleTestCases[] = {
             Instantiation(
                 1,  //
                 L(1, {"foo", "dut", "("}),
-                PortActualList(
-                    3,  //
-                    L(3, {".",          "bus_in", "(",     "{", "brn", "::",
-                          "Num_blocks", "{",      "$bits", "(", "dbg", "::",
-                          "bus_t",      ")",      "'",     "(", "0",   ")",
-                          "}",          "}",      ")",     ","}),
-                    L(3, {".",        "bus_mid", "(",      "{", "brn", "::",
-                          "Num_bits", "{",       "$clog2", "(", "dbg", "::",
-                          "bus_t",    ")",       "'",      "(", "1",   ")",
-                          "}",        "}",       ")",      ","}),
+                PortActualList(3,  //
+                    N(3,  //
+                      L(3, {".", "bus_in", "(", "{", "brn", "::", "Num_blocks", "{"}),
```

Comment:
Shouldn't a new partition level start after `.bus_in(`, with `{...` opening the first call argument?

If this (or any) test is too complicated, feel free to split it into smaller ones that focus on a one or few concepts at a time.

Reply:
I would encourage splitting into smaller focused test cases.  (You can still keep the existing ones.)